### PR TITLE
Add steps for oidc-config and operator-roles

### PR DESCRIPTION
**Pull request by yasun1**

### CODE REVIEW

File: ci-operator/step-registry/rosa/sts/operator-roles/create/rosa-sts-operator-roles-create-commands.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Create operator roles
+echo "Create the operator roles with the prefix ${OPERATOR_ROLES_PREFIX}..."
+rosa create operator-roles -y --mode auto \
+                           --prefix ${OPERATOR_ROLES_PREFIX} \
+                           --oidc-config-id ${oidc_config_id} \
+                           --installer-role-arn ${account_installer_role_arn} \
+                           ${HOSTED_CP_SWITCH} \
+                           ${SHARED_VPC_SWITCH}
+# Store the operator-roles-prefix for the post steps and the operator roles deletion
+echo -n "${OPERATOR_ROLES_PREFIX}" > "${SHARED_DIR}/operator-roles-prefix"
```
Comment (yuwang-RH) [inline ci-operator/step-registry/rosa/sts/operator-roles/create/rosa-sts-operator-roles-create-commands.sh:61]: This step is to create operator-roles prior to cluster spec, is there any other step to create operator roles after cluster spect?

**Comment by yuwang-RH** [comment]
@yasun1 And the account roles and operator roles creation support the channel-group flag which is mainly used to decide the latest supported version. Not sure if we want to support it in this PR, just a reminder

**Comment by yuwang-RH** [comment]
Others LGTM

File: ci-operator/step-registry/rosa/sts/operator-roles/create/rosa-sts-operator-roles-create-commands.sh (hunk 1 above)
Comment (yasun1) [reply ci-operator/step-registry/rosa/sts/operator-roles/create/rosa-sts-operator-roles-create-commands.sh:61]: This is only for byo operator roles. I assume there is no user case with '--cluster' for the operator-roles in Prow CI. We merged it into the cluster provision.

**Pull request merged**